events/beam-robotics.html

```diff
@@ -117,14 +117,16 @@ <h2 class="title text-1.5x">
   </div>
   <div>
     <h2 class="title text-1.5x">
-      Week 4: Coding our Robot
+      Week 4: ASME at UCLA & Robot Assembly
     </h2>
     <p>
-      Robots. Coding. Finally time to code robots?? In this lesson, you will learn how to make your robot move at your
-      command! Once we’ve coded our robot, <a class="tla-link"
-      href="https://www.asmebruins.com/" target="_blank" rel="noopener noreferrer"> ASME at UCLA</a> will come in and
-    teach you a bit about the fancy robots and Battle Bots we make in university.
+      This week, <a class="tla-link"
+      href="https://www.asmebruins.com/" target="_blank" rel="noopener noreferrer">ASME at UCLA</a> will come in and
+    teach you a bit about the fancy robots and Battle Bots we make in university. Next, we'll finish assembling our robots!
```

Comment:
i am sorry for this nit, i feel like `Next` kinda feels like next week since we started off the sentence with `This week,` but feel free to disagree!
```suggestion
    teach you a bit about the fancy robots and Battle Bots we make in university. Then, we'll finish assembling our robots!
```